tasty-inspector/src/scala/tasty/inspector/TastyInspector.scala

```diff
@@ -89,6 +93,19 @@ trait TastyInspector:
 
     end TastyInspectorPhase
 
+    class TastyInspectorFinishPhase extends Phase:
+
+      override def phaseName: String = "tastyInspectorFinish"
+
+      override def run(implicit ctx: Context): Unit =
+        if !alreadyRan then
+          val qctx = QuoteContextImpl()
+          self.finishProcessingCompilationUnits(using qctx)
```

Comment:
What if `finishProcessingCompilationUnits` throws exception? Shound't we use 
`try self.finishProcessingCompilationUnits(using qctx) finally alreadyRan = true` here?

Reply:
I think that this is already handled in the code that orchestrates Phase runs, Tasty Inspector will continue even if `processCompilationUnit` throws an exception and there's no special code for doing that above.

Reply:
My point was not to run `finishProcessingCompilationUnits` twice in such case (there was only `finally` there without `catch`)